templates/components/products/carousel.html

```diff
@@ -40,5 +40,8 @@
     {{/each}}
     {{#if products.length '>' 1}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{products.length}}" class="{{#if customArrowClass}}{{customArrowClass}} {{/if}}js-product-next-arrow slick-next slick-arrow"></button>
+    <span class="carousel-content-change-message aria-description--hidden" role="status">
+        {{lang 'carousel.contentAnnounceMessage'}} {{products.length}}
+    </span>
```

Comment:
can we here also c check slides quantity? because if we have just one slide we do not need this html

Reply:
good point, I've missed it by checking only arrows presentation in theme settings.